thirdeye/thirdeye-frontend/app/pods/components/timeseries-chart/component.js

```diff
@@ -151,6 +156,36 @@ export default Ember.Component.extend({
     this.set('_seriesCache', _.cloneDeep(series));
   },
 
+  /**
+   * Triggers focus or revert when focusedID changes
+   */
+  onFocusChange: Ember.observer('focusedId', function() {
+    const id = this.get('focusedId');
+
+    if (id) {
+      this._focus(id);
+    } else {
+      this._revert();
+    }
+  }),
```

Comment:
we already have a didUpdateAttrs() method. That may be a good place to put all the logic in a single place (and avoid the observer)